File: tuan01/tong.jl
function main()
    data = split(read(stdin, String))
    if isempty(data)
        return
    end
    n = parse(Int, data[1])
    arr = [parse(Int, x) for x in data[2:n+1]]
    
    total_sum = sum(arr)
    max_val = maximum(arr)
    
    println("$total_sum $max_val")
end

main()

File: tuan01/tong.py
import sys

def main():
    data = sys.stdin.read().split()
    if not data:
        return
    n = int(data[0])
    arr = [int(x) for x in data[1:n+1]]
    
    total_sum = sum(arr)
    max_val = max(arr)
    
    print(f"{total_sum} {max_val}")

if __name__ == '__main__':
    main()